app/controllers/organisation_member/shares_controller.rb

```diff
@@ -10,10 +10,7 @@ def index
 
   # GET /shares/1
   def show
-    @official_birth_record = AuditedOperationsService.access_shared_birth_record(
-      logged_identity: current_account,
-      share: @share
-    )
+    @official_birth_record = @share.access(accessed_by: current_account)
```

Comment:
```suggestion
    @official_birth_record = policy_scope @share.access(accessed_by: current_account)
```

Reply:
Will this work before merging #172 ?

Reply:
Just checked this - won't work before merging #172. #172 isn't ready to merge yet and @Br3nda is away, so let's come back to this in #172

Reply:
Yeah, let's add policy scopes later